analyses/tp53_nf1_score/00-tp53-nf1-alterations.R

```diff
@@ -113,12 +114,14 @@ nf1_loss<-cnvConsesus %>%
 tp53_nf1_coding <- tp53_coding %>%
   bind_rows(tp53_loss,nf1_coding,nf1_loss)
 
-# biospecimen IDs for tumor or cell line DNA-seq
+# biospecimen IDs for tumor DNA-seq
 bs_ids <- histology %>%
-  filter(sample_type != "Normal",
+  filter(sample_type == "Tumor",
          experimental_strategy != "RNA-Seq") %>%
+  filter(cohort %in% c("PBTA", "GMKF", "TARGET")) %>%
```

Comment:
We may want to revisit whether we need this filter at all

Reply:
This was just added to match RNA-Seq workflow where we only include these three cohorts for size issues (that part we really need to exclude GTEx). But I think in here, it's fine to remove this filter (or change to `cohort != "GTEx"`?).

Reply:
yes, we can change to only remove gtex